contracts/modules/validators/K1Validator.sol

```diff
@@ -226,25 +225,41 @@ contract K1Validator is IValidator, ERC7739Validator {
     /// @param hash The hash of the data to validate
     /// @param signature The signature data
     function _validateSignatureForOwner(address owner, bytes32 hash, bytes calldata signature) internal view returns (bool) {
-        // Check if the 's' value is valid
```

Comment:
Should I still keep this? @filmakarov

Reply:
Yes.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8b591baef460523e5ca1c53712c464bcc1a1c467/contracts/utils/cryptography/ECDSA.sol#L134-L145

Reply:
ok, adding back

Reply:
done